addons/survey/tests/common.py

```diff
@@ -255,62 +255,64 @@ def _create_one_question_per_type(self):
 
 
 class TestSurveyCommon(SurveyCase):
```

Comment:
Also, please don't put spaces between "task - number" when referencing the task. This can cause github to try to match the number to a commit hash.

Correct syntax is "Task-XXXXX".